cmd/xl-v1.go

```diff
@@ -264,7 +251,5 @@ func getStorageInfo(disks []StorageAPI) StorageInfo {
 // StorageInfo - returns underlying storage statistics.
 func (xl xlObjects) StorageInfo() StorageInfo {
 	storageInfo := getStorageInfo(xl.storageDisks)
-	storageInfo.Backend.ReadQuorum = xl.readQuorum
```

Comment:
Perhaps we can get rid of ReadQuorum checks from server-startup-msg.go and also from the StorageInfo.Backend struct and also please check this with @abperiasamy on the final banner requirement.

Reply:
good catch. updated the banner as discussed with @abperiasamy